scripts/setup.js

```diff
@@ -0,0 +1,39 @@
+import fs from 'fs';
+
+import config from 'config';
+import inquirer from 'inquirer';
+import shell from 'shelljs';
+
+const DATA_REPO = `https://github.com/ambanum/CGUs-data.git`;
+
+(async () => {
+  try {
+    if (fs.existsSync(config.get('history.dataPath'))) {
+      return console.log(`It's seems that the database is already initialized as "${config.get('history.dataPath')}" directory already exists.`);
+    }
+
+    const newHistoryMessage = 'Start a new local fresh history';
+    const downloadHistoryMessage = `Download the entire history of terms of services from ${DATA_REPO}`;
+    const answer = await inquirer.prompt([{
+      type: 'list',
+      message: 'Choose an option to set up the database:',
+      name: 'history',
+      choices: [
+        { name: newHistoryMessage },
+        { name: downloadHistoryMessage }
+      ]
+    }]);
+
+    if (answer.history === newHistoryMessage) {
+      shell.mkdir(config.get('history.dataPath'));
+      shell.cd(config.get('history.dataPath'));
+      shell.exec('git init');
```

Comment:
> Not sure it's a “small” refactor anymore 😅
> 
> My main blocking point has to do with the use of `shelljs` instead of `simple-git` to set up the database. Why go down to that level? 🤔

It was for simplicity, but I changed this.